_docs-2/troubleshooting/system-metadata-tables.md

```diff
@@ -51,12 +51,14 @@ Let's decode this little session:
     key/value pair, and is 186 bytes long.
 
 * `last:13fe86cd27101e5 []    127.0.0.1:9997` -
-    Last location for this tablet.  It was last held on 127.0.0.1:9997, and the
-    unique tablet server lock data was `13fe86cd27101e5`. The default balancer
-    will tend to put tablets back on their last location.
+    Last assigned location for this tablet.  It was last held on 127.0.0.1:9997, and the
+    unique tablet server lock data was `13fe86cd27101e5`.
 
 * `loc:13fe86cd27101e5 []    127.0.0.1:9997` -
-    The current location of this tablet.
+    The last location data was written locally. Data was last written on 127.0.0.1:9997, and the
+    unique tablet server lock data was `13fe86cd27101e5`. The default balancer
+    will tend to put tablets back on their last location where files writes have occurred to 
+    attempt to improve data locality.
```

Comment:
I think these descriptions are backwards. The "last" field is the most recent location written locally, and the "loc" field is the current location (most recently assigned).

I'm not certain about the behavior of the default balancer, but I think at least *some* balancers will use the "last" field in order to try to achieve data locality.